src/js/components/Tab/Tab.js

```diff
@@ -12,194 +12,206 @@ import { normalizeColor } from '../../utils';
 import { StyledTab } from './StyledTab';
 import { TabPropTypes } from './propTypes';
 
-const Tab = forwardRef(
```

Comment:
I changed Tab to use the ref approach [outlined here](https://gist.github.com/gaearon/1a018a023347fe1c2476073330cc5509) instead of using forward ref. It seemed like this was the better approach when a parent has multiple children each with their own ref. So instead of using forwardRef, the refs are passed through TabsContext.

Reply:
I'm fine with the `ref` via context approach. But, I think we still need the `forwardRef` to allow callers to connect a ref to individual Tab components.

Reply:
Fixed in latest commit.